src/Symfony/Component/WebLink/composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "symfony/weblink",
```

Comment:
the confusion is non existent: that's not the same vendor, people do not know about all PSRs
the generic name is also not an issue: the fig already use that name, so it wasn't too generic for them, and again we have the "symfony" prefix, which really suggests "web" to many.

I'm :+1: for going back to just "Link"